muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/chat/ChatPermissions.java

```diff
@@ -0,0 +1,52 @@
+package fi.otavanopisto.muikku.plugins.chat;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import fi.otavanopisto.muikku.model.users.EnvironmentRoleArchetype;
+import fi.otavanopisto.muikku.model.workspace.WorkspaceRoleArchetype;
+import fi.otavanopisto.muikku.security.AbstractMuikkuPermissionCollection;
+import fi.otavanopisto.muikku.security.DefaultEnvironmentPermissionRoles;
+import fi.otavanopisto.muikku.security.MuikkuPermissionCollection;
+import fi.otavanopisto.muikku.security.PermissionScope;
+import fi.otavanopisto.security.Scope;
+
+@ApplicationScoped
+public class ChatPermissions extends AbstractMuikkuPermissionCollection implements MuikkuPermissionCollection {
+
+  @Scope (PermissionScope.ENVIRONMENT)
+  @DefaultEnvironmentPermissionRoles ( { EnvironmentRoleArchetype.ADMINISTRATOR, EnvironmentRoleArchetype.MANAGER, EnvironmentRoleArchetype.STUDY_PROGRAMME_LEADER, EnvironmentRoleArchetype.TEACHER } )
+  public static final String CHAT_CHECKBOX_PERMISSION = "CHAT_CHECKBOX_PERMISSION";
```

Comment:
I admit saying use something like "checkbox-permission" but it could be more descriptive name i.e. what is the permission for or what does it do? :)